vscode-dotnet-runtime-library/src/EventStream/EventStreamEvents.ts

```diff
@@ -260,6 +268,35 @@ export class DotnetFallbackInstallScriptUsed extends DotnetAcquisitionMessage {
     public readonly eventName = 'DotnetFallbackInstallScriptUsed';
 }
 
+export abstract class DotnetFileEvent extends DotnetAcquisitionMessage
+{
+    constructor(public readonly eventMessage: string, public readonly time: string) { super(); }
```

Comment:
one downside to taking the formatted messages as parameters is that you lose what the inputs to the formatted strings were. filepaths, etc go into these messages and so we have a PII concern. Can you look at what it would take to pass in parameters to these constructors and scrub the sensitive fields in `getProperties()`?

Reply:
That's a really good point. Definitely. On it!

Reply:
Done. Also scrubbed the dotnet install script output of paths since that has 'em too.